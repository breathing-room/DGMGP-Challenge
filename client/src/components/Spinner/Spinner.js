import React from 'react';
import Loader from 'react-loader-spinner';

class Spinner extends React.Component {
   render() {
    return(
        <Loader type="BallTriangle" color="rgba(3, 190, 190, 0.651)" height={80} width={80} style={{padding: '40px'}}/>
    );
   }
}

export default Spinner;

// https://www.npmjs.com/package/react-loader-spinner