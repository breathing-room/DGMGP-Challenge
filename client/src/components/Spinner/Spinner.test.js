import React from 'react';
import Spinner from './Spinner'

import { shallow } from 'enzyme'

describe('Spinner component', () => {

  test('it renders correctly', () => {
      const wrapper = shallow(
        <Spinner />
      );
    
      expect(wrapper).toMatchSnapshot();
  });
})

