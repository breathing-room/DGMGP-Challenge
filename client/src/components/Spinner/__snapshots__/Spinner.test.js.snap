// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner component it renders correctly 1`] = `ShallowWrapper {}`;
